docs/en/bridges.wml

```diff
@@ -164,44 +164,59 @@
         "Configure" button.
     </h3>
     <br/>
-    <img src="$(IMGROOT)/tb-tor-launcher-startup.png" alt="Tor Browser's Initial Configuration page">
+    <img src="$(IMGROOT)/bridges/tb-tor-launcher-startup.png"
+      alt="Tor Browser's Initial Configuration page">
     <br><br><br>
-    <h3>2) If you must configure a proxy then select "Yes" and enter the
-        details on the following page.
-        <br/>If you do not use a proxy then select "No" and click "Next".
-        <br/>If you do not know if you must configure a proxy then you likely
-        do not need to do it.
+    <h3>2) Proxy and Censorship prevention settings are now on one page. If
+        your Internet Service Provider (ISP) blocks or otherwise censors
+        connections to the Tor Network, tick this checkbox.
     </h3>
     <br/>
-    <img src="$(IMGROOT)/tb-tor-launcher-no-proxy.png" alt="Tor Browser's Proxy page">
+    <img src="$(IMGROOT)/bridges/tb-tor-launcher-network-settings.png"
+       alt="Tor Browser's Network settings page">
     <br><br><br>
-    <h3>3) After you configure a proxy or skip over that configuration page,
-        the following page asks "Does your Internet Service Provider (ISP)
-        block or otherwise censor connections to the Tor Network?". Select
-        "Yes" and then click "Next".</h3>
+    <h3>3) If you must configure a proxy then, activate the second checkbox and enter the details.
+    </h3>
+    <br/>
+    <img src="$(IMGROOT)/bridges/tb-tor-launcher-proxy.png"
+      alt="Tor Browser's Proxy settings">
+    <br><br><br>
+    <h3>4) Now you have three configuration options. You can use bridges which are
+        preconfigured and provided with Tor Browser, you can specify your own bridge(s), or request bridges.</h3>
     <br/>
-    <img src="$(IMGROOT)/tb-tor-launcher-isp-interference.png" alt="Tor Browser's Bridge page">
+    <img src="$(IMGROOT)/bridges/tb-tor-launcher-bridges.png"
+      alt="Tor Browser's Bridge Configuration page">
     <br><br><br>
-    <h3>4) Now you have two configuration options. You can use bridges which are
-        preconfigured and provided with Tor Browser, or you can specify your
-        own bridge(s).</h3>
+    <h3>5a) If you want to use one of the provided bridges, then choose one of
+        the offered transport types. obfs4 is currently recommend, but
+        depending on where you are located another one may work better for you.
+        If you have any questions, please
+        <a href="<page about/contact>#support">contact us</a>.</h3>
```

Comment:
Redirect users from "Contact us" to Support portal section: https://support.torproject.org/#censorship

Reply:
done.